samples/serverReports/README.md

```diff
@@ -1,4 +1,17 @@
-# server reports for SQL Operations Studio
+# Server Reports for SQL Operations Studio
 
-Welcome to **server reports** for SQL Operations Studio! Server reports give useful insights about the server.
+Welcome to **server reports** for SQL Operations Studio! Server reports give useful insights about the server related to performance that can be used to see current activity as well as historical activity. Here is a list of some of the available reports.
 
+* DB Space Usage
+* DB Buffer Usage
+* CPU Utilization
+* Backup Growth Trend
+* Wait counts
+
+<img src="https://github.com/Microsoft/sqlopsstudio/tree/master/samples/serverReports/images/server_reports.png" alt="server reports" style="width:480px;"/>
+
+It is a sample of dashboard extension. It demonstrates capabilities of the dashboard and the extension. This extension is inspired by SQL Server Management Studio (SSMS)'s reports. We will continually add more useful server insights and tasks.
+
+See [serverReports github repro] for the extension source code.
+
```

Comment:
It would be nice to have "Thank you" section similar to main product repo to thank Paul Randal if we're using some of his scripts.